### DISCUSSION TITLE
Vacation planning (aka multi-day social)

### DISCUSSION THREAD

**Discussion by tarunmohan23** [homostellaris/scheduleyoursocial#16] [Ideas]
Hi it would be super helpful if I could see all inputs into the calendar so I can organise a number a of days for the holiday rather than being given pre defined dates. Is this possible?

**Comment by homostellaris** [comment]
Hi @tarunmohan23, thanks for taking the time to raise an issue, it's really great to get some feedback 🙏

Just want to check I understand correctly first. Rather than seeing a list of individual 'best dates' you'd like to be able to see all the dates everyone picked?

And this is because you are trying to organise an event across several days (a holiday) rather than just one day?

**Comment by tarunmohan23** [comment]
Hi @homostellaris - that’s correct, so essentially being able to see all overlaps with everyone so we can make a decision based on what periods have the most overlaps and a bit of subjectivity. Is that possible?

**Comment by homostellaris** [comment]
That's an interesting use case I hadn't thought of before but it makes sense.

The problem is that I'm working on this by myself as a hobby project so I can only work on one thing at a time and there a few other critical things I want to do with it at the moment.

It would be interesting to get more feedback from other if this is a priority to them so I'm going to convert this to a 'discussion' which means others can upvote it and I can get an idea of interest. You can 'subscribe' to the discussion to see if there are any updates but there is not yet a lot of engagement with this project so just want to set your expectations that realistically I may never got enough feedback to justify working on this (though I am trying to get more engagement!).

Alternatively if you know how to code you could submit a pull request to add this feature yourself?

Either way I really appreciate the feedback and would love more so please mention it to others if you've found this project helpful 🙏

**Reply by homostellaris** [reply]
FYI I've converted this to a discussion now @tarunmohan23, be sure to upvote it.

**Comment by homostellaris** [comment]
Another user [mentioned a similar use case on Reddit](https://www.reddit.com/r/InternetIsBeautiful/comments/v7avxz/comment/l33y44t/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button).

**Comment by homostellaris** [comment]
Brain-dumping some initial thoughts on what I'd change to better support this without compromising too much on simplicity.

- Update the date picker so that the view intelligently updates such that the organisers dates can be seen right away (currently it just shows the current month). This acts as a rough guide for the intended period of the vacation.
- Make it easy to select multiple days or whole weeks in the date picker.
- Provide a view that shows everyone's raw dates. Could be a large calendar view that shows all the emojis next to each day.
- Update the ranked 'best' dates list so that it tries to find consecutive days with the same people and ranks these 'groups' of dates as their own entry.
- Add buttons on the ranked dates so that you can easily 'jump' to those dates in the large calendar view.

**Reply by Baurenz** [reply]
that sound pretty perfect. similar to what i was asking for on reddit. that would be a really cool feature to plan vacation with variable length. havent found any tool which is capable of that and easy to use :) sorry, haven't seen your response earlier.
